src/promptflow/promptflow/_cli/_pf_azure/_run.py

```diff
@@ -130,9 +130,25 @@ def add_parser_run_stream(subparsers):
 
 # Stream run logs:
 pfazure run stream --name <name>
+
+# Stream run logs with timeout:
+pfazure run stream --name <name> --timeout 600
 """
+
+    def add_param_timeout(parser):
+        parser.add_argument(
+            "--timeout",
```

Comment:
timeout-seconds?

Reply:
Offline-synced, will keep the timeout name.